package.json

```diff
@@ -16,10 +16,12 @@
     "webdriver": "webdriver-manager update && webdriver-manager start",
     "wdio-local": "export VERSION=$(node ./scripts/build/get-version.js) && grunt e2e-env-setup && wdio run ./tests/e2e/default/wdio.conf.js",
     "standard-wdio-local": "export VERSION=$(node ./scripts/build/get-version.js) && grunt e2e-env-setup && wdio run ./tests/e2e/standard/wdio.conf.js",
+    "default-wdio-mobile-local": "export VERSION=$(node ./scripts/build/get-version.js) && grunt e2e-env-setup && wdio run ./tests/e2e/default-mobile/wdio.conf.js",
     "e2e-integration": "mocha --config tests/integration/.mocharc.js ",
     "wdio": "wdio run ./tests/e2e/default/wdio.conf.js",
     "upgrade-wdio": "wdio run ./tests/e2e/upgrade/wdio.conf.js",
-    "standard-wdio": "wdio run ./tests/e2e/standard/wdio.conf.js"
+    "standard-wdio": "wdio run ./tests/e2e/standard/wdio.conf.js",
+    "default-wdio-mobile": "wdio run ./tests/e2e/default-mobile/wdio.conf.js"
```

Comment:
BTW there are some wdio files [here](https://github.com/medic/cht-core/tree/master/tests/mobile) but I think these aren't running and I don't know if those are actually fine (like well done and working). 
Do you think we should add those in this PR or a separate one?

Reply:
I go for leave those for a separate PR to keep this one small.

Reply:
I agree, a separate PR sound better 🙂 
We can consider it as part of the organization/restructure that we are working

Reply:
Okay!